### PR TITLE
executor: show warning for `kill pid` statement 

### PR DESCRIPTION
**Pull request by mccxj**
for #5193

### CODE REVIEW

**Comment by shenli** [comment]
@mccxj Thanks!
Please fix CI.

**Comment by mccxj** [comment]
https://github.com/pingcap/tidb/blob/52e7f1a91c722df1fa18fa522bb53ff7fd105fb7/executor/show_test.go#L337

https://github.com/pingcap/tidb/blob/52e7f1a91c722df1fa18fa522bb53ff7fd105fb7/util/testkit/testkit.go#L105

I will fix it. But show_test.go depends on the execution order and number of the test cases, it needs to be improved.

**Comment by mccxj** [comment]
@shenli it is ok now.PTAL

File: executor/simple.go [hunk 1]
```diff
@@ -294,6 +294,9 @@ func (e *SimpleExec) executeKillStmt(s *ast.KillStmt) error {
 			return nil
 		}
 		sm.Kill(s.ConnectionID, s.Query)
+	} else {
+		err := errors.New("Nothing happened. Please use 'KILL TIDB [CONNECTION | QUERY] connectionID' instead")
```
Comment (zz-jason) [inline executor/simple.go]: how about: s/Nothing happened/Invalid operation/ ?

**Comment by shenli** [comment]
LGTM

File: executor/simple.go (hunk 1 above)
Comment (mccxj) [reply executor/simple.go]: OK. PTAL

**Comment by shenli** [comment]
@zz-jason PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**